compiler/src/dmd/backend/blockopt.d

```diff
@@ -2122,3 +2128,22 @@ private void blexit(ref GlobalOptimizer go)
 
     bexits.dtor();
 }
+
+/***********************************
+ * Determine if any addresses of locals on the stack are taken.
+ * Returns:
+ *      true for addresses taken
+ */
+@trusted
```

Comment:
does this really need to be `@trusted`?

Reply:
The function accesses a mutable global variable.